src/js/grapl-cdk/lib/service.ts

```diff
@@ -112,10 +109,10 @@ export class Service {
             description: 'Lambda execution role for: ' + serviceName,
             managedPolicies: [
                 iam.ManagedPolicy.fromAwsManagedPolicyName(
-                    'service-role/AWSLambdaBasicExecutionRole'
+                    'service-role/AWSLambdaBasicExecutionRole' // FIXME: remove managed policy
                 ),
                 iam.ManagedPolicy.fromAwsManagedPolicyName(
-                    'service-role/AWSLambdaVPCAccessExecutionRole'
+                    'service-role/AWSLambdaVPCAccessExecutionRole' // FIXME: remove managed policy
                 ),
```

Comment:
not a blocking request, but a nice-to-have: 
could you add something to the issue tracker, and cross-reference it here?

Reply:
https://github.com/grapl-security/issue-tracker/issues/106